scripts/validate_string_concatenation.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python
+"""
+Check where there is a string that needs to be concatenated.
+"""
+
+import os
+import sys
+import token
+import tokenize
+
+FILE_EXTENTIONS_TO_CHECK = [".py", ".pyx"]
+
+
+def main():
+    path = sys.argv[1]
+
+    if not os.path.exists(path):
+        raise ValueError("Please enter a valid path, to a file/directory.")
+
+    if os.path.isfile(path):
+        # Means that the given path is of a single file.
+        sys.exit(is_concatenated(path))
+
+    status_codes = set()
+    # Means that the given path is of a directory.
+    for subdir, _, files in os.walk(path):
+        for file_name in files:
+            ext = os.path.splitext(os.path.join(subdir, file_name))[1]
+            if ext in FILE_EXTENTIONS_TO_CHECK:
+                status_codes.add(is_concatenated(os.path.join(subdir, file_name)))
+
+    if 1 in status_codes:
+        sys.exit(1)
+
+    sys.exit(0)
+
+
+def is_concatenated(file_path):
```

Comment:
Could add type?

Reply:
At first I typed this whole script, but it said it was a syntax error, same with ```f-strings```, I think that the running platform of this is python 3.5

Reply:
Hmm - we run `validate_docstrings.py` as part of `code_checks.sh` which has 3.6 + syntax?

Reply:
> Hmm - we run `validate_docstrings.py` as part of `code_checks.sh` which has 3.6 + syntax?

I really don't know.

Reply:
> Hmm - we run `validate_docstrings.py` as part of `code_checks.sh` which has 3.6 + syntax?

@alimcmaster1 https://github.com/pandas-dev/pandas/pull/30467#issuecomment-569092377